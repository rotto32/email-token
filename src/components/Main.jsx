import React from 'react';


class Main extends React.PureComponent {
  constructor() {
    super();
  }

  render () {
    return (
      <h1>Test</h1>
    )
  }
}

export default Main;
