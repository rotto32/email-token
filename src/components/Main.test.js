import React from 'react';

import { shallow } from 'enzyme';

import Main from "./Main.jsx";

describe('Main', ()=>{
  it('should render', ()=>{
    const component = shallow( <Main /> );
    expect(component).toMatchSnapshot();
  });

})