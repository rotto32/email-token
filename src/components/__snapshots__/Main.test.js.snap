// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should render 1`] = `ShallowWrapper {}`;
